lambda/exportPass/index.js

```diff
@@ -1,10 +1,10 @@
 const AWS = require('aws-sdk');
 const s3 = new AWS.S3();
 const csvjson = require('csvjson');
-const { runQuery } = require('../dynamoUtil');
+const { runQuery, TIMEZONE } = require('../dynamoUtil');
```

Comment:
Just noticing we have 2 instances of "America/Vancouver" in this dynamoUtil class.

Reply:
Addressed in #116